lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -111,7 +115,7 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
         const appName = changes['appName'];
         if (appName && appName.currentValue) {
             if (this.taskId) {
```

Comment:
we should probably add an and condition that checks the processInstanceId is passed, otherwise the api /$processInstanceId/$taskId will fail